documentation/getting-started/README.md

```diff
@@ -5,44 +5,51 @@ permalink: /documentation/getting-started/
 ---
 # Getting Started
 
-This Getting Started tutorial describes how to record a test, run it and analyze the results.
+This Getting Started tutorial describes how to record a test for the [TestCafe Studio sample page](http://devexpress.github.io/testcafe/example), run the test and analyze the results.
 
-This tutorial uses the [TestCafe Studio sample page](http://devexpress.github.io/testcafe/example) to record the test and consists of the following sections:
+The tutorial consists of the following steps:
 
 * [Install TestCafe Studio](#install-testcafe-studio)
-* [Record Test Actions](#record-test-actions)
+* [Start Recording a Test](#start-recording-a-test)
+* [Interact with the Tested Webpage](#interact-with-the-tested-webpage)
 * [View the Recorded Test](#view-the-recorded-test)
-* [Add New Actions to the Test](#add-new-actions-to-the-test)
+* [Add an Assertion to the Test](#add-an-assertion-to-the-test)
 * [Run the Test](#run-the-test)
 * [View the Test Results](#view-the-test-results)
 
 ## Install TestCafe Studio
 
 You can install TestCafe Studio on MacOS, Windows or Linux. See [Installation](../guides/installation.md) for more information.
 
-## Record Test Actions
+## Start Recording a Test
 
-First, create a directory to store your tests.
-Go to TestCafe Studio, open the **File** menu, click **Open Tests Directory** and navigate to the created directory.
-You can view the directory's contents in the [Explorer](../user-interface/explorer-panel.md) panel.
+When you open TestCafe Stuido at the first time, it shows the **Welcome** page that allows you to start recording a test quickly. Enter the tested site's URL ([http://devexpress.github.io/testcafe/example](http://devexpress.github.io/testcafe/example)) and click the ![Record button](../../images/getting-started/record-test-icon.png) **Start recording a test** button. TestCafe Studio creates a test, launches a browser with the tested site and starts recording your activity on the webpage (see [Interact with the Tested Page](#interact-with-the-tested-page)).
```

Comment:
'Stuido'?
'at the first time'?
'it shows the **Welcome** page that allows you to start recording a test quickly.' Why quickly? -> that allows you to start recording a test.

Reply:
>Stuido 
it's a typo, I'll fix
> at the first time
what's wrong?
>quickly
Yes, the Welcome page allows you to quickly start recording, while a usual approach to start the recording is "go to the Explorer panel, create a test directory and open it, create a fixture, then start recording". On the Welcome page, you don't need to do all of this, only specify the url and click the Record button.

Reply:
There's no such thing as 'at the first time'- it's 'the first time' or 'for the first time'.
Adjectives and adverbs are unnecessary. You do not need to add 'quickly'. You can just state the actions.  

TestCafe Studio displays the **Welcome** page when you open it for the first time. You can specify the URL and click the Record button on this page to start recording a test.

Reply:
ok